warp/builtins.py

```
@@ -4795,41 +4795,72 @@ def compute():
 
 add_builtin(
     "bvh_query_aabb",
-    input_types={"id": uint64, "low": vec3, "high": vec3},
+    input_types={"id": uint64, "low": vec3, "high": vec3, "root": int},
```

Ruff (0.13.1)

`uint64` may be undefined, or defined from star imports

(F405)